README.md

```diff
@@ -40,6 +40,7 @@
 - Test cases for each controller
 - Add views for user and posts
 - Add forms for posts, comments, and likes
+- Add integtation test and fix n+1 problems
```

Comment:
- [ ] In your Posts controller, in the `index` action you need to fetch the comments along with the comments and eliminate N+1 query while doing so. We should have the following

  ```
  @posts = @user.posts.includes(:comments)
  ```
  instead of 
  ```
  @posts = @user.posts
  ```
  ![image](https://github.com/zewdu444/blog_app/assets/36542986/bade3e1f-99f1-458e-91c0-1802f46b69e5)

Reply:
@TedLivist thanks for insightful comment , we have improved based on your feedback